bika/lims/idserver.py

```diff
@@ -156,15 +156,17 @@ def get_variables(context, **kw):
             samplingDate = DT2dt(sampling_date)
         else:
             # No Sample Date?
-            logger.error("Sample {} has no sample date set".format(sample_id))
+            if not context.checkCreationFlag():
```

Comment:
Good catch!

Reply:
I'm happy to be able to sneak this one in.  IMO (@ramonski also), using logger.error should be reserved for the case of unrecoverable error, logger.warning for other cases.  My reasons are selfish - I use Products.PDBDebugMode, and in this case logger.error causes a PDB - sometimes many PDBs in one request.